src/Microsoft.AspNetCore.SignalR.Core/HubConnectionHandler.cs

```diff
@@ -100,7 +100,7 @@ public class HubConnectionHandler<THub> : ConnectionHandler where THub : Hub
             {
                 await DispatchMessagesAsync(connection);
             }
-            catch (Exception ex)
+            catch (Exception ex) when (!(ex is OperationCanceledException))
```

Comment:
Everywhere else we explicitly catch `OperationCanceledException` and don't do anything with it. Should stick with that for consistency (also it gives a nice space for a comment explaining why canceled exception is gnored)

Reply:
Sure